std/haxe/web/Scheme.hx

```diff
@@ -0,0 +1,57 @@
+package haxe.web;
+/**
+* The scheme consists of a sequence of characters beginning with a letter and followed by any combination of letters, digits, plus (+), period (.), or hyphen (-). Although schemes are case-insensitive, the canonical form is lowercase and documents that specify schemes must do so with lowercase letters. It is followed by a colon (:).
+* @class Scheme
+* @module Tamina
+*/
+@:enum abstract Scheme(String) from String to String  {
```

Comment:
I'm not sure this should be an enum because this is far from a complete list of possible values.